git_machete/github.py

```diff
@@ -148,8 +150,22 @@ def create_pull_request(org: str, repo: str, head: str, base: str, title: str, d
         'body': description,
         'draft': draft
     }
-    pr = fire_github_api_get_request('POST', f'/repos/{org}/{repo}/pulls', token, request_body)
-    return parse_pr_json(pr)
+    prs: List[GitHubPullRequest] = derive_pull_requests(org, repo)
+    to_load: GitHubPullRequest = GitHubPullRequest(1, 'user', base, head, '')
+    if not is_pr_already_created(to_load, prs):
+        pr = fire_github_api_get_request('POST', f'/repos/{org}/{repo}/pulls', token, request_body)
+        return parse_pr_json(pr)
+    else:
+        raise MacheteException('Given Pull Request is already created!')
```

Comment:
```suggestion
        raise MacheteException('Pull request for branch {head} is already created!')
```
Plus I'd also maybe display a link to the PR in question (so `is_pr_already_created` maybe should return a `GitHubPullRequest` object instead of a bool)